.editorConfig

```diff
@@ -0,0 +1,6 @@
+# This file is for unifying the coding style for different editors and IDEs	
+# See editorconfig.org
+
+# Enforce line feed at end of line (instead of carriage return + line feed)
+# Prettier doesn't stricly enforce it so we do it here
+end_of_line = lf
```

Comment:
Im not sure if editorconfig plugin is case-sensitive while parsing for file names. But i think `.editorConfig` should be `.editorconfig`

We could probably just remove the redundant styles that prettier would handle there. 

Also, we should keep the configs indicating tabs vs spaces.

https://prettier.io/docs/en/options.html#tabs

We can choose to use prettier to change on run, or we can keep the properties from the last .editorconfig.

Reply:
Prettier enforces by default one indentation = 2 spaces so I didn't include that.

Reply:
Next I want to make travis lint and fix on each pr so that prettier is always enforced.